stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/MessageListView.kt

```diff
@@ -625,6 +629,19 @@ public class MessageListView : ConstraintLayout {
         layoutTransition = LayoutTransition()
     }
 
+    private fun initUnreadLabelButton(unreadLabelButtonStyle: UnreadLabelButtonStyle) {
+        if (unreadLabelButtonStyle.unreadLabelButtonEnabled) {
+            unreadLabelButton = binding.unreadLabelButton
+            unreadLabelButton?.apply {
+                setTextStyle(unreadLabelButtonStyle.unreadLabelButtonTextStyle)
+                backgroundTintList = ColorStateList.valueOf(unreadLabelButtonStyle.unreadLabelButtonColor)
+                rippleColor = ColorStateList.valueOf(unreadLabelButtonStyle.unreadLabelButtonRippleColor)
+            }
+        } else {
+            binding.unreadLabelButton.isVisible = false
+        }
+    }
```

Comment:
```suggestion
    private fun initUnreadLabelButton(unreadLabelButtonStyle: UnreadLabelButtonStyle) {
        binding.unreadLabelButton.isVisible = unreadLabelButtonStyle.unreadLabelButtonEnabled
        if (unreadLabelButtonStyle.unreadLabelButtonEnabled) {
            unreadLabelButton = binding.unreadLabelButton
            unreadLabelButton?.apply {
                setTextStyle(unreadLabelButtonStyle.unreadLabelButtonTextStyle)
                backgroundTintList = ColorStateList.valueOf(unreadLabelButtonStyle.unreadLabelButtonColor)
                rippleColor = ColorStateList.valueOf(unreadLabelButtonStyle.unreadLabelButtonRippleColor)
            }
        }
    }
```

Reply:
Button should only be visible when the first unread message is not shwon.
This property is only to allow to use the button or disable it.